krm-functions/dnn-fn/README.md

```diff
@@ -0,0 +1,28 @@
+# dnn-fn
+
+## Overview
+
+<!--mdtogo:Short-->
+
+`dnn-fn` is a KRM function with two main purposes:
+- for each IP `pool` specified in each `DataNetwork` resource in the kpt package, `dnn-fn` will create a corresponding `IPAllocation`. 
+- for each `IPAllocation` that was created by a previous call of `dnn-fn` it will copy the result of the IP allocation to the `status` of the original `DataNetwork` resource.
+
+<!--mdtogo-->
+
+
+<!--mdtogo:Long-->
+
+## Usage
```

Comment:
Just a suggestion, for the sake of alignment what do you think about changing the content of `usage` sub-section? 

https://github.com/nephio-project/nephio/tree/main/krm-functions/interface-fn#usage
https://github.com/nephio-project/nephio/blob/main/krm-functions/ipam-fn/README.md
https://github.com/nephio-project/nephio/blob/main/krm-functions/nad-fn/README.md#usage

This section can be names as Implementation details, perhaps?